learn/update_and_migration/previous_docs_version.md

```diff
@@ -0,0 +1,84 @@
+# Accessing previous documentation versions
+
+Meilisearch's documentation only covers the latest stable release. It is possible, however, to consult the documentation of previous Meilisearch versions stored in our Git repository.
```

Comment:
I've been actively trying to remove terms like "now", "currently", and "for the moment" from the docs. I think they don't really add much information (beyond a vague "this might change" vibe) and most sentences become clearer without it. 

I like your suggestion otherwise, though—may I remove that bit?

Reply:
Sure! I edited it for ya 🙆🏻